src/main/resources/ValidationMessages.properties

```diff
@@ -1,2 +1,5 @@
 javax.validation.constraints.NotBlank.message=This information must be provided
-javax.validation.constraints.NotNull.message=This information must be provided
\ No newline at end of file
+javax.validation.constraints.NotNull.message=This information must be provided
+uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory.message=Efficiency rating is not within range
+uk.gov.beis.els.api.common.ApiValuesFromRatingClassRange.message=Efficiency rating is not within range
```

Comment:
Given its not always the efficiency rating (e.g. noise emission class) and the value may not be a valid range at all, think it would be better to say `Value is invalid or not within allowed range`

Reply:
done